PaddleAudio/examples/dcase2021_task1b/README.md

```diff
@@ -0,0 +1,111 @@
+# DCASE challenge 2021 task1b, Audio-visual scene classification
```

Comment:
Why pre-commit not block the pickle files to commit? Commit large files to github repo is not a recommended usage.

Reply:
fixed